cmd/nodecfg/apply.go

```diff
@@ -52,6 +53,9 @@ func init() {
 	applyCmd.Flags().StringVarP(&nodeConfigBucket, "bucket", "b", "", "S3 bucket to get node configuration from.")
 
 	applyCmd.Flags().BoolVarP(&disableDNS, "disable-dns", "N", false, "disable setting DNS entries")
+
+	applyCmd.Flags().BoolVarP(&skipMetricsDNS, "skip-metrics-dns", "M", false, "do not set metrics DNS entries")
```

Comment:
is this tool mostly used for large private network testing? if so, maybe the default should be true.

Reply:
nodecfg is used for all algonet recipes and only mmet is a large enough were we want to skip, so not skipping by default looks reasonable for me